util/conversion/conversion.go

```diff
@@ -17,11 +17,16 @@ limitations under the License.
 package conversion
 
 import (
+	"context"
 	"math/rand"
+	"strings"
 	"testing"
 
 	fuzz "github.com/google/gofuzz"
 	"github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
This import caused me to double take...  not sure why they aren't exposing this out via k8s.io/api/apiextensions/<version>...

Reply:
Related-ish https://github.com/kubernetes/client-go/issues/247